llvm/lib/Target/AArch64/AArch64CallingConvention.cpp

```diff
@@ -210,7 +210,7 @@ static bool CC_AArch64_Custom_Block(unsigned &ValNo, MVT &ValVT, MVT &LocVT,
   }
 
   const Align StackAlign =
-      State.getMachineFunction().getDataLayout().getStackAlignment();
+      *State.getMachineFunction().getDataLayout().getStackAlignment();
```

Comment:
This should probably try to handle the missing case?

Reply:
It shouldn't be missing. I'll add an assert (here and in other places).